content/internships/contents.lr

```diff
@@ -43,67 +43,16 @@ If you're interested in this program, please review the resources below and come
 
 Here are our mentors for this round of internships:
 
-<table class="table table-sm table-striped">
-  <thead class="thead-light">
-    <tr>
-      <th>Name</th>
-      <th>Slack username</th>
-      <th>About</th>
-    </tr>
-    <tr>
-      <td>Alden Page</td>
-      <td><code>@aldenpage</code></td>
-      <td><a href="/blog/authors/aldenpage">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Anna Tumadóttir</td>
-      <td><code>@Anna</code></td>
-      <td><a href="https://creativecommons.org/author/annacreativecommons-org/">See CC staff bio</a></td>
-    </tr>
-    <tr>
-      <td>Ari Madian</td>
-      <td><code>@Ari</code></td>
-      <td><a href="/blog/authors/akmadian">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Breno Ferreira</td>
-      <td><code>@brenoferreira</code></td>
-      <td><a href="/blog/authors/brenoferreira">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Brent Moran</td>
-      <td><code>@Brent</code></td>
-      <td><a href="/blog/authors/mathemancer">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Dhruv Bhanushali</td>
-      <td><code>@dhruvkb</code></td>
-      <td><a href="/blog/authors/dhruvkb">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Hugo Solar</td>
-      <td><code>@hugo</code></td>
-      <td><a href="/blog/authors/hugosolar">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Kriti Godey</td>
-      <td><code>@kriti</code></td>
-      <td><a href="/blog/authors/kgodey">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>María Belén Guaranda Cabezas</td>
-      <td><code>@Maria Belen Guaranda</code></td>
-      <td><a href="/blog/authors/soccerdroid">See CC Open Source Blog author page</a></td>
-    </tr>
-    <tr>
-      <td>Sarah Hinchcliff Pearson</td>
-      <td><code>@shinchpearson</code></td>
-      <td><a href="https://creativecommons.org/author/sarahcreativecommons-org/">See CC staff bio</a></td>
-    </tr>
-    <tr>
-      <td>Timid Robot Zehta</td>
-      <td><code>@Timid Robot</code></td>
-      <td><a href="/blog/authors/TimidRobot">See CC Open Source Blog author page</a></td>
-    </tr>
-  </thead>
-</table>
+| Name                          | Slack Username          | About                                                                                       |
+| ----------------------------- | ----------------------- | ------------------------------------------------------------------------------------------- |
+| Alden Page                    | `@aldenpage`            | <a href="/blog/authors/aldenpage">See CC Open Source Blog author page</a>                   |
```

Comment:
Please use markdown syntax for the links instead of HTML.

Reply:
@kgodey 
updated, please review
Thanks